### PR TITLE
fix: hidden scroll if extension is empty

### PR DESCRIPTION
**Pull request by Lisofffa**

### CODE REVIEW

File: src/contentscript/overlay/root/pages/Dapplets/index.tsx [hunk 1]
```diff
@@ -291,7 +291,12 @@ export const Dapplets: FC<DappletsProps> = (props) => {
       {isLoadingListDapplets ? (
         <TabLoader />
       ) : (
-        <div className={cn(styles.dappletsBlock, classNameBlock)}>
+        <div
+          className={cn(styles.dappletsBlock, classNameBlock, {
+            [styles.noScroll]:
+              isNoContentScript || !filteredDapplets || filteredDapplets.length === 0,
+          })}
```
Comment (alsakhaev) [inline src/contentscript/overlay/root/pages/Dapplets/index.tsx:298]: A scroll must be displayed only when container is overfilled and must be implemented using only CSS, no JavaScript. The current implementation doesn't work when number of elements > 0 and container is not overfilled:
image

**Review by alsakhaev — CHANGES_REQUESTED** [review]

File: src/contentscript/overlay/root/pages/Dapplets/index.tsx (hunk 1 above)
Comment (Lisofffa) [reply src/contentscript/overlay/root/pages/Dapplets/index.tsx]: +

**Comment by alsakhaev** [comment]
Perfect!

**Pull request merged**